app/models/Login.java

```diff
@@ -0,0 +1,42 @@
+package models;
+
+import play.data.Form;
+
+/**
+ * Created by lapa on 5/18/17.
+ */
+public class Login {
+
+    public static Form<Login> loginForm = Form.form(Login.class);
```

Comment:
Add form validations (required...) on this model, and check form for errors in login controller.